### PR TITLE
Adding README from Speedometer project

### PR DESCRIPTION
**Pull request by mdwelsh**

### CODE REVIEW

**Comment by quietbamboo** [comment]
OK, seems Matt has done that.
So shall I just merge this pull request?

**Comment by mdwelsh** [comment]
Ping? Can someone review this and merge?

File: README [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+details).
+
+IMPORTANT NOTE:
+
+This code was originally developed as an internal measurement tool by
+Google. We decided to open source it to support the mobile networking
+research community. As such, in its current form, the code is designed
+to work on Google's internal AppEngine infrastructure, and will need
+some work to remove the Google dependencies before it can be used more
+broadly. In particular the app checks for a "google.com" account on
```
Comment (gavaletz) [inline README]: @drchoffnes -- correct me if I am wrong, but this should be already taken care of right?  We have removed the need for a google.com address, replaced it with the need for a gmail.com compatible address and it should build and deploy from an external GAE instance.

File: README [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+Code Reviews
+------------
+
+All pull requests on github will be subject to code review. Code reviews
+are an excellent way to ensure that any new code is designed well and
+that we can catch bugs early.
+
+All new code should adhere to the Google Style Guides:
+
+Python: http://code.google.com/p/google-styleguide/
```
Comment (gavaletz) [inline README]: Would it be better to point to the actual style guide?

http://google-styleguide.googlecode.com/svn/trunk/pyguide.html

**Comment by gavaletz** [comment]
Other than those 2 minor things, LGTM.

File: README (hunk 1 above)
Comment (drchoffnes) [inline README]: You are correct.

**Pull request merged**